src/TesApi.Web/AzureProxy.cs

```diff
@@ -729,11 +729,11 @@ private static Task<string> GetAzureAccessTokenAsync(string resource = "https://
         /// Gets an authenticated Azure Client instance
         /// </summary>
         /// <returns>An authenticated Azure Client instance</returns>
-        private static async Task<Azure.IAuthenticated> GetAzureManagementClientAsync()
+        private static async Task<Microsoft.Azure.Management.Fluent.Azure.IAuthenticated> GetAzureManagementClientAsync()
```

Comment:
There is a namespace "Azure" in Azure.Core package, so needed to fully qualify here. Fluent management is going away BTW.